utils/sendMails.js

```diff
@@ -0,0 +1,78 @@
+const nodemailer = require('nodemailer');
+const handlebars = require('handlebars');
+const fs = require('fs');
+const { join } = require('path');
+
+const config = require('../config');
+const TokenService = require('../src/token.service');
+const { TOKEN_TYPE } = require('../constant');
+const { BadRequestError } = require('./customError');
+
+async function sendEmail(opt) {
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    port: 587,
+    secure: true,
+    auth: {
+      type: 'OAUTH2',
```

Comment:
We should ___refactor___ this so that we do not use hardcoded values on lines `13` and `17`